src/store/lokapi.ts

```diff
@@ -46,6 +47,39 @@ export function lokapiStoreFactory(lokApiService: any, gettext: any) {
         }
         dispatch("setupAfterLogin")
       },
+      async signup(
+        { commit, dispatch }: any,
+        credentials: {
+          login: string
+          lastname: string
+          firstname: string
+          password: string
+          confirm_password: string
+        }
+      ) {
+        const { login, lastname, firstname, password, confirm_password } =
+          credentials
+        commit("auth_request")
+        try {
+          await lokApiService.signup(
+            login,
+            lastname,
+            firstname,
+            password,
+            confirm_password
+          )
+        } catch (err: any) {
+          commit("auth_error")
```

Comment:
This also seems strange. You seem to mix `login` and `signup` here. These are disconnected process, aren't they ?.